### ISSUE TITLE
Звуки формата "rolling_**" на 81-714

### ISSUE THREAD

**Issue by pinok21** [metrostroi-repo/MetrostroiAddon#409]
Стоит добавить звуки формата rolling_** и промежуточным вагонам, как реализовано на головных вагонах. У Е-образных это давно реализовано, для реализма было бы и хорошо, чтобы у номерного была такая же фишка.

**Comment by Hellss** [comment]
Вроде есть они (по крайней мере на версии гитхаба)

**Comment by pinok21** [comment]
> Вроде есть они (по крайней мере на версии гитхаба)

да, сами звуки есть, но эти звуки проигрываются только в головных вагонах
